src/dev/impl/DevToys/Views/Tools/Converters/NumberBaseConverter/NumberBaseConverterToolPage.xaml.cs

```diff
@@ -47,21 +47,22 @@ protected override void OnNavigatedTo(NavigationEventArgs e)
                 DataContext = ViewModel;
             }
 
-            if (!string.IsNullOrWhiteSpace(parameters.ClipBoardContent))
-            {
-                string clipBoardContent = NumberBaseFormatter.RemoveFormatting(parameters.ClipBoardContent).ToString();
+            // TODO: This feature would be nice to find a way to keep it
+            //if (!string.IsNullOrWhiteSpace(parameters.ClipBoardContent))
+            //{
+            //    string clipBoardContent = NumberBaseFormatter.RemoveFormatting(parameters.ClipBoardContent).ToString();
 
-                if (NumberBaseHelper.IsValidBinary(clipBoardContent!))
-                {
-                    ViewModel.InputBaseNumber = NumberBaseFormat.Binary;
-                }
-                else if (NumberBaseHelper.IsValidHexadecimal(clipBoardContent!))
-                {
-                    ViewModel.InputBaseNumber = NumberBaseFormat.Hexadecimal;
-                }
+            //    if (NumberBaseHelper.IsValidBinary(clipBoardContent!))
+            //    {
+            //        ViewModel.InputBaseNumber = NumberBaseFormat.Binary;
+            //    }
+            //    else if (NumberBaseHelper.IsValidHexadecimal(clipBoardContent!))
+            //    {
+            //        ViewModel.InputBaseNumber = NumberBaseFormat.Hexadecimal;
+            //    }
 
-                ViewModel.InputValue = parameters.ClipBoardContent;
-            }
+            //    ViewModel.InputValue = parameters.ClipBoardContent;
+            //}
```

Comment:
It would be good to revert this changes and keep this feature. This could require big changes but if you give me any idea I will try to fix it.